Concept Anchor - Formal pattern improvements (#530)
gruetter/innerSourceTest: patterns/1-initial/concept-anchor.md

## Resulting Context

> Management and newcomers to InnerSource engage in conversations around InnerSource, percieved benefits, expected difficulties, etc. Usually comparing to  previous experiences around the anchored concepts.

Reviewer: ```suggestion
Management and newcomers to InnerSource engage in conversations around InnerSource, perceived benefits, expected difficulties, etc.
They are usually comparing InnerSource to previous experiences around the anchored concepts.
```

Splitting into two lines.
Fix typo.
Extend a bit.

## Status

> Initial.
> These are ideas for solutions to the problem that have not been tested yet.

Reviewer: ```suggestion
* Initial

The above are ideas for solutions to the problem that have not been tested yet.
```

> This is still a pattern any Known Instances.

Reviewer: ```suggestion
This pattern does not have any Known Instances yet.
```

I wasn't sure what you meant here.

Tried to reword it. Did you mean that?
